packages/page/src/Application/MessageHandler/CreatePageMessageHandler.php

```diff
@@ -62,4 +50,15 @@ public function __invoke(CreatePageMessage $message): PageInterface
 
         return $page;
     }
+
+    public function setParent(string $parentId, PageInterface $page): PageInterface
```

Comment:
should be private or?

```suggestion
    private function setParent(string $parentId, PageInterface $page): PageInterface
```